backend/models/playlist.js

```diff
@@ -18,35 +18,34 @@ class Playlist {
         words.length > 4 ? (title = words.slice(0, 4).join(" ")) : (title = book.title)
         //use first 4 words of the title
 
-        const url = `https://api.spotify.com/v1/search?q=${title}&type=playlist&limit=3`
-        const titleResponse = await axios.get(url, {
-            headers: {
-              'Authorization': `Bearer ${token}`
-            }
-          });
+        let [titleResponse, genreResponse] = 
```

Comment:
Thank you for adding in these optimizations! This should help reduce the loading time significantly :)